applications/appunorganized/appunorganized-plugins/org.csstudio.service.math/src/org/csstudio/service/math/AddServiceMethod.java

```diff
@@ -2,31 +2,29 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.function.Consumer;
 
-import org.epics.pvmanager.WriteFunction;
-import org.epics.pvmanager.service.ServiceMethod;
-import org.epics.pvmanager.service.ServiceMethodDescription;
-import org.epics.vtype.VNumber;
+import org.diirt.service.ServiceDescription;
+import org.diirt.service.ServiceMethod;
+import org.diirt.service.ServiceMethodDescription;
+import org.diirt.vtype.VNumber;
 
 public class AddServiceMethod extends ServiceMethod {
 
-    public AddServiceMethod() {
-    super(new ServiceMethodDescription("add", "Adds two numbers")
-        .addArgument("arg1", "First argument", VNumber.class)
-        .addArgument("arg2", "Second argument", VNumber.class)
-        .addResult("result", "The sum", VNumber.class));
+    public AddServiceMethod(ServiceMethodDescription serviceMethodDescription, ServiceDescription serviceDescription) {
```

Comment:
@shroffk changes I made to math service.  This seems like an example, should we just remove this from the repo.